modules/skparagraph/src/ParagraphCache.cpp

```diff
@@ -123,21 +123,23 @@ uint32_t ParagraphCacheKey::computeHash() const {
         if (ts.fStyle.isPlaceholder()) {
             continue;
         }
-        hash = mix(hash, SkGoodHash()(relax(ts.fStyle.getLetterSpacing())));
-        hash = mix(hash, SkGoodHash()(relax(ts.fStyle.getWordSpacing())));
-        hash = mix(hash, SkGoodHash()(ts.fStyle.getLocale()));
-        hash = mix(hash, SkGoodHash()(relax(ts.fStyle.getHeight())));
-        hash = mix(hash, SkGoodHash()(relax(ts.fStyle.getBaselineShift())));
+        hash = mix(hash, SkGoodHash()(ts.fStyle.getFontStyle()));
```

Comment:
Not only. 9 valuus instead of 5 now. The order now matches fields declaration to avoid missing something, I beleive it's more important that potential merge conflicts

Reply:
The best guarantee would be a test. But we don't run skia tests (perhaps we should). A test in skiko might be an alternative, but I suppose ParagraphCacheKey is not exposed in skiko right now?

Reply:
> I suppose ParagraphCacheKey is not exposed in skiko right now?

And should not, it's not a part of the public API